fe/fe-core/src/main/java/com/starrocks/sql/optimizer/ChildOutputPropertyGuarantor.java

```diff
@@ -244,6 +256,17 @@ public Void visitPhysicalMergeJoin(PhysicalMergeJoinOperator node, ExpressionCon
         return visitPhysicalJoin(node, context);
     }
 
+    @Override
+    public Void visitPhysicalNestLoopJoin(PhysicalNestLoopJoinOperator node, ExpressionContext context) {
```

Comment:
nest loop join could use required property to generate broadcast property, This class is to guarantee child output property is legal.

Reply:
done